IBAnimatable/MaskType.swift

```diff
@@ -4,12 +4,68 @@
 //
 
 import Foundation
+import UIKit
 
-public enum MaskType: String {
+/// Add a masking shape.
+public enum MaskType: IBEnum {
+  
+  /// Circle shape with diameter equals to min(width, height)
   case circle
-  case polygon
-  case star
+  
+  ///  Polygon shape with n sides. (min 3, default)
+  case polygon(sides:Int?)
+  /// Star shape with n points (min 3, default:6)
+  
+  case star(points:Int?)
+  /// Isocele triangle shape. The triangle's height is equal to the view's frame height. If the view is a square, Triangle is equilateral.
   case triangle
-  case wave
-  case parallelogram
+  
+  /// Wave shape, with
+  /// direction (up or down, default : up)
+  /// width (default 40)
+  /// offset (default : 0)
+  case wave(direction:WaveDirection?, width:Float?, offset:Float?)
```

Comment:
Yes, All the MaskType documentation is shown in the Xcode 8 quick view documentation ;)